Components/Rating/rating.css

```diff
@@ -0,0 +1,35 @@
+.rating-container {
+    display: flex;
+    flex-direction: column;
+    gap: var(--container-gap);
+    background-color: var(--box-color);
+    width: var(--container-width);
+    height: 15%;
+    justify-content: center;
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+}
+
+.star-section {
+    display: flex;
+    background-color: var(--bg-color);
+    gap: 0.25rem;
```

Comment:
suggestion: this should be in px. value too small to be a relative unit

Reply:
Done